### PR TITLE
Add many programs

### PR DESCRIPTION
**Pull request by aandrew-me**
Added the following programs:
vhs
draw
gambit
slides
linuxwave
zramd
muffon
bauh
ytmdl
pulsar
navi
gpt4all

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +107,10 @@ _For a more comprehensive/advanced/better categorized/... list of Linux audio so
 - [![Open-Source Software][oss icon]](https://gitlab.gnome.org/World/lollypop) [Lollypop](https://wiki.gnome.org/Apps/Lollypop) - A GNOME music playing application.
 - [![Open-Source Software][oss icon]](https://gitlab.com/ColinDuquesnoy/MellowPlayer) [Mellow Player](https://colinduquesnoy.gitlab.io/MellowPlayer/) - Cloud music integration for your desktop.
 - [![Open-Source Software][oss icon]](https://github.com/mopidy/mopidy) [Mopidy](https://www.mopidy.com/) - An extensible music server written in Python.
+- [![Open-Source Software][oss icon]](https://github.com/staniel359/muffon) [muffon](https://muffon.netlify.app/) - muffon is a cross-platform music streaming browser
```
Comment (DimitrisPa) [inline README.md:110]: Please reduce the extra lines!

Comment (aandrew-me) [reply README.md:110]: Oh right

Comment (aandrew-me) [reply README.md]: Fixed

File: README.md [hunk 2]
```diff
@@ -1448,6 +1457,7 @@ _For a more comprehensive/advanced/better categorized/... list of Linux audio so
 - [![Open-Source Software][oss icon]](https://github.com/iamtalhaasghar/yewtube) [yewtube](https://github.com/iamtalhaasghar/yewtube) - Terminal based YouTube player and downloader. No Youtube API key required. Forked from mps-youtube. Can be installed using pip as described [here](https://github.com/iamtalhaasghar/yewtube#using-pip).
 - [![Open-Source Software][oss icon]](https://github.com/rg3/youtube-dl) [youtube-dl](https://rg3.github.io/youtube-dl/) - youtube-dl is a command-line program to download videos from YouTube.com and a few more sites. It requires the Python interpreter (2.6, 2.7, or 3.2+), and it is not platform specific.
 - [![Open-Source Software][oss icon]](https://github.com/yt-dlp/yt-dlp) [yt-dlp](https://github.com/yt-dlp/yt-dlp) - A youtube-dl fork with additional features and fixes.
+- [![Open-Source Software][oss icon]](https://github.com/deepjyoti30/ytmdl) [ytmdl](https://github.com/deepjyoti30/ytmdl) - A simple app to get songs from YouTube in mp3 format with artist name, album name etc from sources like iTunes, Spotify, LastFM, Deezer, Gaana etc.
```
Comment (DimitrisPa) [inline README.md:1457]: The project has a website https://ytmdl.deepjyoti30.dev/. Can you please include it?

Comment (aandrew-me) [reply README.md:1457]: I didn't include it because the website is not about the project but a tool by itself

Comment (DimitrisPa) [reply README.md:1457]: Oh, I see. Ok we can skip it then. Thanks for the PR btw.

**Review by DimitrisPa — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (aandrew-me) [reply README.md:1457]: Happy to Contribute!

**Pull request merged**